### DISCUSSION TITLE
Request: add instructions to serve documentation locally

### DISCUSSION THREAD

**Discussion by DeflateAwning** [zauberzeug/nicegui#3304] [Ideas / Feature Requests]
It would be awesome if you could add instructions to serve the documentation locally. It should go in either the README, in the CONTRIBUTING.md file, or maybe both.

**Comment by python-and-novella** [comment]
By running `pip install poetry`,you will get `poetry`.Then you could clone repo to empty dir.
After you cloned repo,in repo's root,run `poetry install` firstly,then run `poetry run python main.py`.
It is very simply.

**Comment by rodja** [comment]
The quickest way may be `docker run -p 8080:8080 zauberzeug/nicegui`. I created #3305 to add this to the docs.

**Reply by DeflateAwning** [reply]
Awesome, thanks a lot!
